application/lib/service/offers_screen_service.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:application/model/offer_dto.dart';
+import 'package:cloud_firestore/cloud_firestore.dart';
+
+class OffersScreenService {
+  final FirebaseFirestore _db = FirebaseFirestore.instance;
+
+  Future<List<OfferDTO>> fetchOffers({bool orderByViewCount = false}) async {
+    Query query = _db.collection('offers');
+    if (orderByViewCount) {
```

Comment:
A boolean változók neve melléknév szokott lenni és is-el vagy has-el szoktuk kezdeni, ezzel is jelezve azt, hogy logikai értéket tartalmaz, így nem kell megnézni a deklarálását ahhoz, hogy tudjuk. Kérlek legyen a neve isOrderRequiredByViewCount